data/src/test/java/org/apache/iceberg/data/avro/TestGenericData.java

```diff
@@ -76,4 +86,29 @@ protected void writeAndValidate(Schema writeSchema, Schema expectedSchema) throw
   protected boolean supportsDefaultValues() {
     return true;
   }
+
+  @Test
+  public void testSchemaWithTwoVariants() throws JsonProcessingException {
```

Comment:
Why is this test in `TestGenericData`? It looks like this is testing the output of `convert(schema, "table")` but passing the schema through `GenericData.Record` and a `GenericRecordBuilder` first?

Reply:
I was trying to have test coverage for two variant columns in one Avro schema. Since it can be covered with testVariantConversion(), I can remove this one.